haskoin-core/Network/Haskoin/Script/Types.hs

```diff
@@ -53,7 +53,7 @@ instance Binary Script where
             empty <- isEmpty
             if empty
                 then return []
-                else liftM2 (:) get getScriptOps
+                else (pure (:)) <*> get <*> getScriptOps
```

Comment:
Can you change it for the simpler version using `(<$>)`? I will merge when you do and @plaprade gives his ack. Consider this pull request "acked" by me.